CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGE] Federation: Only use local TSDB for federation (ignore remote read). #7077
+* [FEATURE] Tracing(!): Added experimental Jaeger support #7148
+* [ENHANCEMENT] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
+* [ENHANCEMENT] TSDB: Significantly reduce how much old WAL we keep around. #7098
+* [BUGFIX] Discovery: Add `architecture` meta label for EC2. #7000
+* [BUGFIX] React UI: Fixed multiselect legend on OSX. #6880
+* [BUGFIX] Remote Write: Fixed blocked resharding edge case. #7122
+* [BUGFIX] Remote Write: Fixed the edge case of not updating remote write on relabel configs change. #7073
```

Comment:
I don't think saying this is an edge case adds anything. `Fixed remote write not updating on relabel configs change` is what we did.

Reply:
:+1: